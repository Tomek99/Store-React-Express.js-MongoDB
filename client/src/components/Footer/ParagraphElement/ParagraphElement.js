import React from "react";
import styles from "./ParagraphElement.module.scss";

function ParagraphElement({ text, icon }) {
  return (
    <div className={styles.ParagraphElement}>
      <p>
        {icon}
        {text}
      </p>
    </div>
  );
}

export default ParagraphElement;
