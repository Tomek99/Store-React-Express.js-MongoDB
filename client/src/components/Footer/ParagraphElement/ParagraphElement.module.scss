.ParagraphElement {
  p {
    display: flex;
    align-items: center;
    gap: 5px;
    color: #282828;
  }
}
