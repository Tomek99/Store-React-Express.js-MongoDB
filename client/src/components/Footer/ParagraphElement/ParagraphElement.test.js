import { render } from "@testing-library/react";
import ParagraphElement from "./ParagraphElement";

test("should render ParagraphElement component", () => {
  const data = { name: "test" };
  const icon = "testIcon";

  const { container } = render(
    <ParagraphElement text={data.name} icon={icon} />
  );

  expect(container).toMatchSnapshot();
});
