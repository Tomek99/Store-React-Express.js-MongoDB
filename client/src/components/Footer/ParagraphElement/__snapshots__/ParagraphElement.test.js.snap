// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ParagraphElement component 1`] = `
<div>
  <div
    class="ParagraphElement"
  >
    <p>
      testIcon
      test
    </p>
  </div>
</div>
`;
